src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1207,32 +1208,119 @@ void missingRdfTypeOnNamedNode() {
         }
     }
 
+    class BaseExample {
+        Model model;
+        Resource system;
+        ValidateAIF validator = seedlingValidator;
+
+        void setup() {
+            model = createModel();
+            system = makeSystemWithURI(model, getTestSystemUri());
+        }
+
+        void addType(Resource resource, Resource type) {
+            markType(model, getAssertionUri(), resource, type, system, 1d);
+        }
+
+        void testInvalid(String name) {
+            assertAndDump(model, name, validator, false);
+        }
+
+        void testValid(String name) {
+            assertAndDump(model, name, validator, true);
+        }
+
+    }
+
     /**
      * Set of tests to show that NIST restrictions pass and fail appropriately
      */
     @Nested
-    class NISTExamples {
-        Model model;
-        Resource system;
+    class NISTExamples extends BaseExample {
         Resource entity;
         Resource event;
+        Resource relation;
         Resource entityCluster;
         Resource eventCluster;
+        Resource relationCluster;
+        NISTExamples() { validator = nistSeedlingValidator; }
```

Comment:
NISTExamples could @override `testValid` and `testInvalid` to prepend `"NIST.valid"` or `"NIST.invalid"` so we could remove it from the calls to `testValid` and `testInvalid` and promote uniformity and avoid copy-paste errors.

Just a suggestion for your consideration-- I don't feel strongly.

Reply:
I think this is a good idea, but I think it should probably be incorporated into the next refactoring of this file. Ideally, we'd like to split this file up and I think I'd like the implementation of this idea to have the individual tests extend from a AbstractBaseTestSet or something common that would run testValid/Invalid. The concrete subclasses could then be responsible for producing a valid and invalid model. We can work out the details later, but let me know what you think

Reply:
I don't have a problem with it being part of the refactoring you describe (which I like), but sometimes refactoring tasks get set aside (not necessarily on AIDA but in general).  Would you object to it being written up as a JIRA task and set as a blocker for [AIDA-611](https://nextcentury.atlassian.net/browse/AIDA-611) (which is currently in our next Sprint)?  That means the new issue would need to go in the next Sprint also.

Reply:
Depends on when we get to a stable ExampleAndValidationTest.java. I would like to avoid refactoring the single file into multiples while there are several people working on it at once. That being said, it _should_ be a ticket and blocking 611 is a really good place for it.

Reply:
Added [AIDA-698](https://nextcentury.atlassian.net/browse/AIDA-698) to capture this.